src/app/debug/table/table.component.ts

```diff
@@ -213,12 +217,35 @@ export class TableComponent implements OnInit, OnDestroy {
   loadReportInProgressSettings() {
     this.httpService.getSettings().subscribe({
       next: (settings) => {
-        this.tableSettings.reportsInProgress = settings.reportsInProgress;
+        this.tableSettings.numberOfReportsInProgress = settings.reportsInProgress;
         this.tableSettings.estimatedMemoryUsage = settings.estMemory;
+        this.loadReportInProgressDates();
       },
     });
   }
 
+  loadReportInProgressDates() {
+    let hasChanged: boolean = false;
+    for (let index = 1; index <= this.tableSettings.numberOfReportsInProgress; index++) {
```

Comment:
Report with index 0 gets skipped?

Reply:
this is done because in backend, the api does the following for some odd reason 
`Report report = testTool.getReportInProgress(index - 1);`

So that would have to be fixed first.

Reply:
In a way, it is logical. When asking "how many reports are in progress", 0 would mean there are none in progress. 
But thinking of it as an array filled with reports that are in progress, it would be more logical if the first item is 0. The length of the array is the amount of reports in progress.